bgpd/bgp_route.c

```diff
@@ -5165,7 +5165,9 @@ void bgp_update(struct peer *peer, const struct prefix *p, uint32_t addpath_id,
 	 * attr->evpn_overlay with evpn directly. Instead memcpy
 	 * evpn to new_atr.evpn_overlay before it is interned.
 	 */
-	if (soft_reconfig && evpn && afi == AFI_L2VPN) {
+	if (evpn && afi == AFI_L2VPN &&
```

Comment:
Wouldn't this just be enough like:

```c
if (evpn) {
 ...
}
```

Reply:
Okay, but... If the AFI is L2VPN, then how is this going to be configured `!CHECK_FLAG(peer->af_flags[afi][safi], PEER_FLAG_SOFT_RECONFIG)`? Under `l2vpn evpn` AF?